services/api/axios.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { DEV_BASE_URL } from '../../config/constants';
+
+export const axiosInstance = axios.create({
+  baseURL: DEV_BASE_URL,
+  headers: {
+    Authorization: 'Bearer ZfbR9xctgNBRH5kSAqppbnwkvoJMarYdI2tf2yElGCitSaiAVQX5twHK8BWYjNLG9NNqqLv2FQ12ZTEB',
```

Comment:
🤯

Reply:
@antosan what's the best way to handle this?

Reply:
I'm considering adding a `.env.example` file and the updating the documentation steps

Reply:
First of all, since we have already leaked these credentials, I would ask the BE team to revoke them and generate new ones.
Using a `.env` file is how I would do it. One thing though is that we should not put the actual secrets in the `.env.example` file since that is committed to the repo. We should only put the keys there and dummy values.

```
EXPO_PUBLIC_API_AUTHORIZATION_KEY="fake-key"
EXPO_PUBLIC_DEV_BASE_URL="https://dev.droidcon.co.ke/api/v1"
EXPO_PUBLIC_PROD_BASE_URL="https://api.droidcon.co.ke/v1"
```

Can you also update the CONTRIBUTING guide to include the fact that to run the app locally you need to fill in the actual values after copying the `.env.example` file to `.env.development`.

I also noticed that the API returns the data even when the `Authorization` header is not set. I believe we only need to pass the `Api-Authorization-Key` header only. Can you confirm this? If this is the case, we could remove the `EXPO_PUBLIC_TOKEN` from the `.env` file. The less secrets we have in the `.env` file the better.

The rest depends on the usecase, I believe we do not handle any personal information in this API and therefore prefixing the key with `EXPO_PUBLIC_` is OK. Otherwise I would not do that since these variables will be visible in plain-text in your compiled application.